core/server/web/shared/middlewares/labs.js

```diff
@@ -1,21 +1,15 @@
 const labsUtil = require('../../../services/labs');
 const common = require('../../../lib/common');
 
-const labs = {
-    subscribers(req, res, next) {
-        if (labsUtil.isSet('subscribers') === true) {
-            return next();
-        } else {
-            return next(new common.errors.NotFoundError());
-        }
-    },
-    members(req, res, next) {
-        if (labsUtil.isSet('members') === true) {
-            return next();
-        } else {
-            return next(new common.errors.NotFoundError());
-        }
+const labs = flag => (req, res, next) => {
```

Comment:
```suggestion
const labs = (flag) => (req, res, next) => {
```

Just for consistency, afair we are using brackets around all multiline arrow functions :)

Reply:
This needs updating in eslint config if we want it to be consistent

Reply:
Yeah think so, sorry this was long time ago where it was decided in a PR/Issue and idea was to update the config at some point later after using, but never happened. I'll try to find it and re-raise it i guess. :)

Reply:
No probs :) I just hate spending time on fixing lint stuff, so if it's in eslint it will get fixed automatically :relaxed: